index.html

```diff
@@ -0,0 +1,554 @@
+                <p>Corrections</p>
+            </div>
+            <div class="section">
+                <p class="section-name">OPINION</p>
+                <p>Todays's Opinion</p>
+                <p>Op-Ed Columnists</p>
+                <p>Editorials</p>
+                <p>Op-Ed Contributors</p>
+                <p>Letters</p>
+                <p>Sunday Review</p>
+                <p>Video: Opinion</p>
+            </div>
+            <div class="section">
+                <p class="section-name">ARTS</p>
+                <p>Today's Arts</p>
+                <p>Art & Design</p>
+                <p>Books</p>
+                <p>Dance</p>
+                <p>Movies</p>
+                <p>Music</p>
+                <p>Pop Culture</p>
+                <p>Television</p>
+                <p>Theater</p>
+                <p>Video:Arts</p>
+            </div>
+            <div class="section">
+                <p class="section-name">LIVING</p>
+                <p>At Home</p>
+                <p>Automobiles</p>
+                <p>Games</p>
+                <p>Education</p>
+                <p>Food</p>
+                <p>Health</p>
+                <p>Jobs</p>
+                <p>Love</p>
+                <p>Magazine</p>
+                <p>Parenting</p>
+                <p>Real Estate</p>
+                <p>Recipies</p>
+                <p>Style</p>
+                <p>T Magazine</p>
+                <p>Travel</p>
+            </div>
+            <div class="section">
+                <p class="section-name">MORE</p>
+                <p>Reader Center</p>
+                <p>Wirecutter</p>
+                <p>Live Events</p>
+                <p>The Learning Network</p>
+                <p>Tools & Services</p>
+                <p>Multimedia</p>
+                <p>Photography</p>
+                <p>Video</p>
+                <p>Newsletters</p>
+                <p>TimesMachine</p>
+                <p>NYT Store</p>
+                <p>Times Journeys</p>
+                <p>Manage My Account</p>
+            </div>
+            <div class="section">
+                <div class="sub-section">
+                    <p class="section-name">SUBSCRIBE</p>
+                    <div class="sub-sections">
+                        <p>Home Delivety</p>
+                        <p>Digital Subscriptions</p>
+                        <p>Games</p>
+                        <p>Cooking</p>
+                    </div>
+                    <div class="sub-types">
+                        <p>Email Newsletters</p>
+                        <p>Corporate Subscriptions</p>
+                        <p>Education Rate</p>
+                    </div>
+                    <div class="app-types">
+                        <p>Mobile Applications</p>
+                        <p>Replica Edition</p>
+                        <p>Espa;ol</p>
+                        <p>中文网</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
```

Comment:
- [ ] all of these should be links ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))